CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to ChainerCV
+
+:tada: Thank you for your interest in contributing to ChainerCV :tada:
+
+We welcome any kind of contributions to ChainerCV!
+
+Feel free to submit a PR or raise an issue even if the related tasks are not currently supported by ChainerCV.
+ChainerCV is intended to include any codes that make computer vision research and development easier with Chainer.
```

Comment:
I heard that "code" is basically uncountable. So I recommend to change "codes" to "code" here.